ResearchKit/Common/ORKAnswerFormat.h

```diff
@@ -250,6 +259,21 @@ ORK_CLASS_AVAILABLE
 
 @end
 
+
+
+/**
+ The `ORKVerticalScaleAnswerFormat `class represents an answer format that includes a vertical slider control.
+ 
+ The vertical scale answer format produces an `ORKScaleQuestionResult` object that contains an integer whose
+ value is between the scale's minimum and maximum values, and represents one of the quantized step values.
+ */
+ORK_CLASS_AVAILABLE
+@interface ORKVerticalScaleAnswerFormat : ORKScaleAnswerFormat
+
+@end
```

Comment:
Would it be much simpler to just add a `vertical` boolean property on  `ORKScaleAnswerFormat` and `ORKContinuousScaleAnswerFormat `?